worldedit-core/src/main/java/com/sk89q/worldedit/world/storage/TrueZipMcRegionChunkStore.java

```diff
@@ -73,20 +74,16 @@ public TrueZipMcRegionChunkStore(File zipFile) throws IOException, ZipException
         zip = new ZipFile(zipFile);
     }
 
-    /**
-     * Get the input stream for a chunk file.
-     *
-     * @param name      the name
-     * @param worldName the world name
-     * @return an input stream
-     * @throws IOException   if there is an error getting the chunk data
-     * @throws DataException if there is an error getting the chunk data
-     */
```

Comment:
Where did this go to?

Reply:
It's on an override method. There were already javadocs on the super method